healthcare/api-client/v1/consent/README.rst

```diff
@@ -12,12 +11,14 @@ This directory contains samples for Cloud Healthcare API. `Cloud Healthcare API`
 
 
 
-.. _Cloud Healthcare API: https://cloud.google.com/healthcare/docs
+.. _Cloud Healthcare API: https://cloud.google.com/healthcare-api/docs
 
 To run the sample, you need to enable the API at: https://console.cloud.google.com/apis/library/healthcare.googleapis.com
 
-To run the sample, you need to have `Healthcare Consent Store Administrator` role.
 
+To run the sample, you need to have the following roles:
+* `Healthcare Consent Store Administrator`
+* `Healthcare Attribute Definition Editor`
```

Comment:
These show up all bundled in the same line, I think you might need an extra space?